be/src/column/stream_chunk.h

```diff
@@ -0,0 +1,105 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include "column/chunk.h"
+#include "column/chunk_extra_data.h"
+
+namespace starrocks::vectorized {
+
+using Int8ColumnPtr = std::shared_ptr<vectorized::Int8Column>;
+
+enum StreamRowOp : std::int8_t { INSERT = 0, DELETE = 1, UPDATE_BEFORE = 2, UPDATE_AFTER = 3 };
+using StreamRowOps = std::vector<StreamRowOp>;
```

Comment:
can we use FixedLengthColumn<StreamRowOp> instead of std::vector<StreamRowOp>?
use StreamRowOpColumn instead of StreamRowOps?

Reply:
Good advise~ However I found `StreamRowOps ` is no used anywhere, I just remove it.